packages/fe-container/src/components/Composer.tsx

```diff
@@ -5,14 +5,31 @@ import {LoadingAnimation} from '@mia-platform/microlc-ui-components'
 import viewEngine from '../composer/ViewEngine'
 import useConfiguration from '../hooks/useConfiguration'
 
+const requiredStringShape = PropTypes.shape({
+  type: PropTypes.string.isRequired
+})
+
+const currentUserShape = PropTypes.shape({
+  properties: PropTypes.shape({
```

Comment:
This is not the shape of user: is the shape of its json schema.

Only the content of properties is needed.

Reply:
made a mistake... but the problem remains.

```javascript
const currentUserShape = PropTypes.shape({
  avatar: PropTypes.string,
  email: PropTypes.string.isRequired,
  groups: PropTypes.arrayOf(PropTypes.string.isRequired),
  name: PropTypes.string.isRequired,
  nickname: PropTypes.string
}).isRequired

const propTypes = {
  configurationName: PropTypes.string.isRequired,
  currentUser: currentUserShape
}
```

if we use this proptype we get into conflicts due to the fact that the absence of .isRequired is equivalent to 'string | null | undefined'.

the removal of nulls was the reason behind using a plain object and 
```javascript
type ComposerProps = PropTypes.InferProps<typeof propTypes> & {
  currentUser?: object
}
```